types/block.go

```diff
@@ -202,6 +206,7 @@ func (h *Header) StringIndented(indent string) string {
 %s  Data:           %v
 %s  Validators:     %v
 %s  App:            %v
+%s  Version:	    %v
```

Comment:
formatting

Reply:
hrm, looks ok here https://github.com/tendermint/tendermint/blob/495-version-in-block-header/types/block.go#L209 and also locally:
![screenshot from 2017-10-12 08-27-56](https://user-images.githubusercontent.com/8304391/31495886-45cd0dfc-af27-11e7-95ae-30e86631e913.png)

am I missing something?

Reply:
@zramsay you have a tab in there, if my assumption is that you are using vim in Ubuntu, type `:set list` and you'll see the tab right after `Version:^I`, of
which we aren't using tabs in the entire format string, please remove it and manually line it up with spaces. Once done, `:set nolist` to return to normal.

Reply:
thanks, fixed